packages/pigeon/lib/dart_generator.dart

```diff
@@ -11,6 +11,9 @@ import 'ast.dart';
 import 'functional.dart';
 import 'generator_tools.dart';
 
+/// Documentation comment open symbol.
+const String openDoc = '///';
```

Comment:
Nit: How about "docCommentPrefix"? "Open" implies that there should be a "close", which isn't the case for this comment style.